protocol-units/bridge/move-modules/sources/atomic_bridge.move

```diff
@@ -222,7 +227,8 @@ module atomic_bridge::atomic_bridge_counterparty {
        complete_bridge_transfer(
```

Comment:
Question about this comment (which I previously wrote)
`// Client must be a moveth minter, otherwise this will fail`, this is no longer true now right?

Would be great if you could write another test, that explicitly calls `complete_bridge_transfer` from a non-moveth minter, to prove this all works

Reply:
Right, removed that comment.

Because `complete_bridge_transfer` includes this:

```
        // Make caller a minter of MovETH
        moveth::add_minter(master_minter, signer::address_of(caller));
```

that means the test `test_complete_transfer_assets` uses a non-moveth minter now.

I removed the line in `test_complete_transfer_assets` where the non-moveth minter is made a minter, before calling `complete_bridge_transfer` so now `test_complete_transfer_assets` can be 

The `client` in `test_complete_transfer_assets` becomes `caller` in `complete_bridge_transfer` and gets made into a minter pending reveling the hashlock preimage.